src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow
```

Comment:
I chatted with @zpao and his feedback is files shouldn’t be partially flow-ified. Otherwise we’ll forget to go back and flowify the rest of their code. New files can have `@flow` as long as they are 100% flowified.